spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -231,6 +231,19 @@
         post :create, user: { email: user.email.upcase, password: user.password }
       end
 
+      it 'caches unverified PII pending confirmation' do
+        user = create(:user, :signed_up)
+        create(
+          :profile,
+          deactivation_reason: :verification_pending,
+          user: user, pii: { ssn: '1234' }
+        )
+
+        post :create, user: { email: user.email.upcase, password: user.password }
+
+        expect(controller.user_session[:decrypted_pii]).to match '1234'
```

Comment:
does the cache of the PII indicate that it's unverified? is there a `verified: false` we should include?